test/integration/targets/hostname/tasks/Debian.yml

```diff
@@ -16,5 +16,5 @@
 - name: Test DebianStrategy using assertions
   assert:
     that:
-      - "'{{ ansible_distribution_release }}-bebop.ansible.example.com' in get_hostname.stdout"
-      - "'{{ ansible_distribution_release }}-bebop.ansible.example.com' in grep_hostname.stdout"
+      - "ansible_distribution_release ~ '-bebop.ansible.example.com' in get_hostname.stdout"
+      - "ansible_distribution_release ~ '-bebop.ansible.example.com' in grep_hostname.stdout"
```

Comment:
What's the Debian related change here? Or is this just a syntax cleanup?

Reply:
It's code that likely worked in the past, but throws an error in current ansible releases. It went unnoticed because CI doesn't test against Debian (only Ubuntu).

Reply:
What error do you get without this change?

Reply:
The error I get is:
```
TASK [hostname : Test DebianStrategy using assertions] *************************
[WARNING]: conditional statements should not include jinja2 templating
delimiters such as {{ }} or {% %}. Found: '{{ ansible_distribution_release
}}-bebop.ansible.example.com' in get_hostname.stdout
fatal: [testhost]: FAILED! => {"msg": "The conditional check ''{{ ansible_distribution_release }}-bebop.ansible.example.com' in get_hostname.stdout' failed. The error was: Conditional is marked as unsafe, and cannot be evaluated."}
```

A small caveat: I get this error in release 2.18.1, not against devel.

Reply:
I noticed that I only get this error on 2.18.1 and not on devel, so I bisected it. #83922 introduces the regression. I assume it's not failing on devel because there's no data tagging in place yet.